Bio/phenotype/phen_micro.py

```diff
@@ -112,18 +109,29 @@ class PlateRecord(object):
     H01
     H02
     H03
-    ...
+    H04
+    H05
+    H06
+    H07
+    H08
+    H09
+    H10
+    H11
+    H12
 
     All the wells belonging to a "column" (identified by the number of the well)
     in the plate can be obtained:
 
     >>> for well in plate.get_column(12):
     ...     print(well.id)
-    ...
```

Comment:
Can you put this ``...`` line back, you would see it at the interactive python prompt.